msalautomationapp/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
-def msalVersion = "5.3.0"
+def msalVersion = "5.+"
```

Comment:
5.3.1*

Technically shouldn't matter going into the release branch, since we're done with ui automation